triage/models/dsapp_scaler.org

```diff
@@ -0,0 +1,570 @@
+* DsappCutOff
```

Comment:
It is documentation. (Literate programming, basically) It shows the use of the class, and keeps syncronized with the code.